features/abi_apk_splits.feature

```diff
@@ -3,73 +3,89 @@ Feature: Plugin integrated in project with ABI APK splits
-    And the field "versionCode" for multipart request 8 equals "2"
+    And the request is valid for the Android Mapping API
+    And the field "versionCode" for multipart request equals "2"
+    And I discard the oldest request
 
-    And the request 9 is valid for the Android Mapping API
-    And the field "versionCode" for multipart request 9 equals "3"
+    And the request is valid for the Android Mapping API
+    And the field "versionCode" for multipart request equals "3"
+    And I discard the oldest request
 
-    And the request 10 is valid for the Android Mapping API
-    And the field "versionCode" for multipart request 10 equals "4"
+    And the request is valid for the Android Mapping API
+    And the field "versionCode" for multipart request equals "4"
+    And I discard the oldest request
 
-    And the request 11 is valid for the Android Mapping API
-    And the field "versionCode" for multipart request 11 equals "5"
+    And the request is valid for the Android Mapping API
+    And the field "versionCode" for multipart request equals "5"
+    And I discard the oldest request
 
-    And the request 12 is valid for the Android Mapping API
-    And the field "versionCode" for multipart request 12 equals "6"
+    And the request is valid for the Android Mapping API
+    And the field "versionCode" for multipart request equals "6"
+    And I discard the oldest request
 
-    And the request 13 is valid for the Android Mapping API
-    And the field "versionCode" for multipart request 13 equals "1"
+    And the request is valid for the Android Mapping API
+    And the field "versionCode" for multipart request equals "1"
+    And I discard the oldest request
 
-    And the request 14 is valid for the Android Mapping API
-    And the field "versionCode" for multipart request 14 equals "7"
+    And the request is valid for the Android Mapping API
+    And the field "versionCode" for multipart request equals "7"
+    And I discard the oldest request
 
-    And the request 15 is valid for the Android Mapping API
-    And the field "versionCode" for multipart request 15 equals "8"
-    And the field "apiKey" for multipart request 15 equals "TEST_API_KEY"
-    And the field "versionName" for multipart request 15 equals "1.0"
-    And the field "appId" for multipart request 15 equals "com.bugsnag.android.example"
+    And the request is valid for the Android Mapping API
+    And the field "versionCode" for multipart request equals "8"
+    And the field "apiKey" for multipart request equals "TEST_API_KEY"
+    And the field "versionName" for multipart request equals "1.0"
+    And the field "appId" for multipart request equals "com.bugsnag.android.example"
 
 @skip_agp4_1_or_higher
 Scenario: ABI Splits automatic upload disabled
     When I build "abi_splits" using the "all_disabled" bugsnag config
+    And I wait for 5 seconds
     Then I should receive no requests
 
 @skip_agp4_1_or_higher
 @skip_agp3_5
```

Comment:
Several scenarios are being skipped currently due to the payload ordering issue.  That will be addressed in the PR mentioned above.